config/holes.go

```diff
@@ -60,8 +60,8 @@ type Hole struct {
 	DataMap                     ordered.Map    `json:"-"`
 	Experiment                  int            `json:"-"`
 	ID                          string         `json:"id"`
-	ItemDelimiter               string         `json:"-" toml:"item-delimiter"`
-	MultisetDelimiter           string         `json:"-" toml:"multiset-delimiter"`
+	MultisetItemDelimiter       string         `json:"-" toml:"multiset-item-delimiter"`
+	OutputDelimiter             string         `json:"-" toml:"output-delimiter"`
```

Comment:
I feel like these renames, while perhaps clearer, has nothing to do with custom judges. Could it be pulled out of this PR?